.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,48 @@
-Before submitting a pull request, make sure it's targeting the right branch:
+<!--- --------------------------------------------------------------------- -->
+<!---                 Please fill the following template                    -->
+<!---             Your pull request may be ignored otherwise                -->
+<!--- --------------------------------------------------------------------- -->
 
-- For the latest version, use `develop`.
-- For bug fixes incompatible with `develop`, use the support branch of the latest compatible and supported version `support/6.x`.
+## Description
+<!--- Describe your changes in detail. Include any relevant information     -->
+<!--- (reasons, difficulties, links to web references, related issues...)   -->
 
-If you're fixing a JavaScript issue, it would help to create a new test case under the folder `test/visual/` that recreates the issue and show's that it's been fixed. Run `npm test` to compile the testing folder.
 
-Happy coding! :)
+
+<!--- Bugs and new features/improvements must be presented and discussed in -->
+<!--- an issue first. Please create one if there is no issue related to     -->
+<!--- this pull request.                                                    -->
+- Closes <!--- link to issue... -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve?              -->
+
+## Screenshots (if appropriate):
+
+## Types of changes
+<!--- What types of changes does your code introduce?                       -->
+<!--- Put an `x` in all the boxes that apply:                               -->
+- [ ] Documentation
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing
+      functionality to change)
+
+## Checklist (all required):
+<!--- Go over all the following points, and put an `x` in the boxes.        -->
+<!--- If you're unsure about any of these, don't hesitate to ask.           -->
+- [ ] I have read and follow the [CONTRIBUTING](CONTRIBUTING.md) document.
+- [ ] There is no other pull request similar to this one.
+- [ ] The pull request title is descriptive.
+- [ ] The template is fully and correctly filled.
+- [ ] The pull request target the right branch (`develop` or `support/*`).
+- [ ] My commits are correctly titled and contains all relevant information.
+- [ ] My code follows the code style of this project.
```

Comment:
Is the code style written down somewhere so we can link to it?

Reply:
Yeah. There is "Coding Standards" in `CONTRIBUTING.md`